server/.env.template

```diff
@@ -1,7 +1,10 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Zeeshan, Saad, Liang"
 dbUser=
 dbPassword=
+<<<<<<< HEAD
```

Comment:
Merge conflict statements, remember to remove

Reply:
😅😅😅😅😅😅😅